src/objects/task2.js

```diff
@@ -0,0 +1,13 @@
+const academy = {
+    title: 'devMentor',
+    course: 'fundamentals',
+    mentorship: true
+}
+
+console.log(academy.title)
+console.log(academy['course'])
+
+const propertyName = ['mentorship']
+
```

Comment:
Missing dynamic field access example: print to the console mentorship field, using const value (that was initialized above).